man/systemd-boot.xml

```diff
@@ -651,7 +651,7 @@
 
     <para>
       <command>qemu-system-x86_64 <replaceable>[ ... ]</replaceable>
-      -kernel /usr/lib/systemd/boot/efi/systemd-bootx64.efi</command>
+      -kernel /usr/lib/systemd/boot/systemd-bootx64.efi</command>
```

Comment:
I doubt we can do this. It's become part of well-known paths and might be hardcoded in various other projects.

Reply:
We definitely can't do this, mkosi relies on the path being /usr/lib/systemd/boot/efi

Reply:
yeah we have scripts relying on this too

Reply:
Well, first of all, a compat symlink is provided, so the old path will still work. And yes, we can rename things with enough announcement and forethought. We shouldn't keep every historical naming decision.

Reply:
The old directory makes little sense. There's an additional subdir inserted that is not used for anything.